changelogs/current.yaml

```diff
@@ -29,6 +29,9 @@ minor_behavior_changes:
 - area: http2
   change: |
     Metadata is parsed with the QUICHE HPACK library, rather than nghttp2. This behavior change can be reverted by setting ``envoy.reloadable_features.http2_decode_metadata_with_quiche`` to ``false``.
+- area: upstream
+  change: |
+    Upstream stream does not disable reading (in case where downstream buffer reaches high watermark) till the full response headers have been received. This behavior change can be reverted by setting ``envoy.reloadable_features.upstream_wait_for_response_headers_before_disabling_read`` to ``false``.
```

Comment:
Changed HTTP/1 upstream streams to not disable reading....

Reply:
received. This fixes a bug where Envoy timeouts were not correctly adjusting to the fact that the response has been sent from upstream.  This behavior change...

Reply:
done